docs/upload-your-snap.md

```diff
@@ -0,0 +1,55 @@
+# Upload to the store
+
+So you've been working hard on your .snap, and you finally have it to the point
+where you're ready to share it with the world? Great! You can use Snapcraft to
+upload it directly to the store.
+
+
+## Authenticate to the store
+
+First of all, you need to give Snapcraft permission to communicate with the
+store on your behalf:
+
+    $ snapcraft login
+    Enter your Ubuntu One SSO credentials.
+    Email: me@example.com
+    Password:
+    OTP: <press enter here if you haven't enabled two-factor authentication>
+    Authenticating against Ubuntu One SSO.
+    Starting new HTTPS connection (1): login.ubuntu.com
+    Login successful.
+
+These credentials will remain valid until you revoke them, which you can do
+with the following:
+
+    $ snapcraft logout
+    Clearing credentials for Ubuntu One SSO.
+    Credentials cleared.
+
+Now, let's upload that .snap!
+
+
+## Upload the .snap
+
+Get into the directory containing the `snapcraft.yaml`, and do the following:
+
+    $ snapcraft upload
```

Comment:
Possibly... it's a nice shortcut. Not sure how useful it'll be unless you're certain it'll built properly